.travis.yml

```diff
@@ -2,3 +2,10 @@ language: node_js
 node_js:
     - "node"
     - "lts/*"
+services:
+  - postgresql
+env:
+  - TESTING_ENVIRONMENT=1
```

Comment:
Travis-CI has a default `CI=true` environment variable: https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables (and so does [Circle-CI](https://circleci.com/docs/2.0/env-vars/#build-details)), if we don't want to create/manage another env variable. Or, we could set `process.env.NODE_ENV=test` or something.